### PR TITLE
[Ubuntu] Add MS SQL Tools version

### PR DESCRIPTION
**Pull request by vsafonkin**
# Description
Added MS SQL Tools version (sqlcmd tool) into software report.
Removed from MySQL report section.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1 [hunk 1]
```diff
@@ -18,18 +18,33 @@ function Get-MySQLVersion {
     return "MySQL $mySQLVersion"
 }
 
+function Get-sqlcmdVersion {
```
Comment (AlenaSviridenko) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1:21]: maybe `Get-SQLCmdVersion`?

**Review by AlenaSviridenko — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Comment by miketimofeev** [comment]
@vsafonkin maybe create Database tools section and move it there along with the **MS SQL Server Client Tools**?
cc @AlenaSviridenko @maxim-lobanov

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Comment by maxim-lobanov** [comment]
> @vsafonkin maybe create Database tools section and move it there?
> cc @AlenaSviridenko @maxim-lobanov

I guess we already have it

File: images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1 (hunk 1 above)
Comment (vsafonkin) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Databases.psm1]: Done, thanks!

**Comment by miketimofeev** [comment]
@maxim-lobanov I mean like we did on windows
https://github.com/actions/virtual-environments/blob/7b54aa465e602330e50f75c06c7c63a72b42617e/images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1#L166-L173

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**